src/safe_apps/tests/test_signals.py

```diff
@@ -193,10 +363,28 @@ def test_on_tag_create_hook_call(self) -> None:
                 responses.matchers.json_params_matcher({"invalidate": "Chains"}),
             ],
         )
+        responses.add(
+            responses.POST,
+            "http://alternative.cgw.url/v2/flush",
+            status=200,
+            match=[
+                responses.matchers.header_matcher(
+                    {"Authorization": "Basic alternative-token"}
+                ),
+                responses.matchers.json_params_matcher({"invalidate": "Chains"}),
+            ],
+        )
 
         Tag().save()  # create
 
-        assert len(responses.calls) == 1
+        assert len(responses.calls) == 2
+        assert isinstance(responses.calls[0], responses.Call)
+        assert responses.calls[0].request.body == b'{"invalidate": "Chains"}'
+        assert responses.calls[0].request.url == "http://127.0.0.1/v2/flush"
+        assert (
+            responses.calls[0].request.headers.get("Authorization")
+            == "Basic example-token"
+        )
```

Comment:
Same here

Reply:
Also fixed in [44d59c7](https://github.com/safe-global/safe-config-service/pull/900/commits/44d59c7c8a3b8845a22ae7d355f56139dd979f5e)